src/stream/pipeline/v4l_pipeline.rs

```diff
@@ -78,9 +78,11 @@ impl V4lPipeline {
                     concat!(
                         "v4l2src device={device} do-timestamp=true",
                         " ! videoconvert",
-                        " ! capsfilter name={filter_name} caps=video/x-raw,format=I420,width={width},height={height},framerate={interval_denominator}/{interval_numerator}",
+                        " ! x264enc",
+                        " ! h264parse",
+                        " ! capsfilter name={filter_name} caps=video/x-h264,stream-format=avc,alignment=au,width={width},height={height},framerate={interval_denominator}/{interval_numerator}",
```

Comment:
I believe that we can accomplish the same thing with decodebin3 here, and letting it taking decision over the encoders. @joaoantoniocardoso what you think ?